d3a_interface/utils.py

```diff
@@ -427,3 +428,28 @@ def scenario_representation_traversal(sc_repr, parent=None):
             yield from scenario_representation_traversal(child, sc_repr)
 
     yield sc_repr, parent
+
+
+class HomeRepresentationUtils:
+    @staticmethod
+    def _is_home(representation):
+        home_devices = ["PV", "Storage", "Load", "MarketMaker"]
+        return all("type" in c and c["type"] in home_devices
+                   for c in representation["children"])
```

Comment:
we should check if `children` key exists in here instead of making sure before calling it

Reply:
It is already checked in the is_home_area method, however I could move the checks in the _is_home method. Doing it